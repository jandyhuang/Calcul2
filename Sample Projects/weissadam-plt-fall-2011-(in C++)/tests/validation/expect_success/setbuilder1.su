function B [int b] returns int
{ 

 int i;
 set a = { 0.0, 0.5, 1.0, 1.5 };
 set c = { "a", "b", "c" };
 set C = {(d,e) | d in { "a", "b", "c" }, e in { "a", "b", "c" }};
 set D = {("a", "a"), ("b", "b"), ("c", "c")};

 if ( C == D ) then
 {
   return 0;
 }
 else
 {
   return 1;
 } 

}
