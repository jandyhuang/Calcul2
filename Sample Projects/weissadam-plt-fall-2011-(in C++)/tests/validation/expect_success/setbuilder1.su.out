c set_seq
i int
C set_seq
a set_seq
D set_seq
b int
